### PR TITLE
docs typo coverage

### PR DESCRIPTION
**Pull request by Eomm**
The 100% club

### CODE REVIEW

**Comment by Eomm** [comment]
Uncovederd lines:

```
 ObjectSchema.js     |      100 |       98 |      100 |      100 |               311 |
 utils.js            |      100 |    95.45 |      100 |      100 |            49,120 |
```

I will try to add them 👍

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by climba03003** [comment]
ObjectSchema#L311 conditional args
```js
it('empty definitions', () => {
  expect(
    ObjectSchema({ generateIds: true })
      .definition('entity')
      .prop('prop', S.ref('entity'))
      .valueOf()
  ).toEqual({
    definitions: {
      entity: {
        $id: '#definitions/entity',
      },
    },
    properties: {
      prop: {
        $ref: 'entity',
      },
    },
    type: 'object',
  })
})
```

**Comment by mcollina** [comment]
@climba03003 what do you mean?

**Comment by climba03003** [comment]
> 
> 
> @climba03003 what do you mean?

Test Case for missing branches. I do not know how to commit in PR. So, I leave it here for reference.

**Comment by mcollina** [comment]
@climba03003 maybe you can send a fresh PR with that included. I think @Eomm is going to take some time off OSS.

File: README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ A fluent API to generate JSON schemas (draft-07) for Node.js and browser. Framew
 - Fluent schema implements JSON Schema draft-07 standards
 - Faster and shorter way to write a JSON Schema via a [fluent API](https://en.wikipedia.org/wiki/Fluent_interface)
 - Runtime errors for invalid options or keywords misuse
-- JavaScript constants can be used in the JSON schema (e.g. _enum_, _const_, _default_ ) avoiding discrepancies between model and schema
-- TypeScript definitions
-- Coverage 99%
+- Javascript constants can be used in the JSON schema (e.g. _enum_, _const_, _default_ ) avoiding discrepancies between model and schema
```
Comment (Eomm) [inline README.md:16]:
```suggestion
- JavaScript constants can be used in the JSON schema (e.g. _enum_, _const_, _default_ ) avoiding discrepancies between model and schema
```

File: README.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ A fluent API to generate JSON schemas (draft-07) for Node.js and browser. Framew
...
+- Typescript definitions
```
Comment (Eomm) [inline README.md:17]:
```suggestion
- TypeScript definitions
```

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by aboutlo** [comment]
builds are failing because % Branch isn't 100%

**Pull request closed without merging**